src/sentry/api/urls.py

```diff
@@ -197,9 +196,6 @@
         AuthenticatorIndexEndpoint.as_view(),
         name='sentry-api-0-authenticator-index'),
 
-    # Sudo
-    url(r'^sudo/$', SudoEndpoint.as_view(), name='sentry-api-0-sudo'),
```

Comment:
Assuming nothing is referencing `sentry-api-0-sudo` as a named route that will break.